### DISCUSSION TITLE
Bring the Fullscreen mode to the Beta Project

### DISCUSSION THREAD

**Discussion by matheus-kirchesch-old** [community/community#14092] [Projects and Issues] [answered]
I like to download projects as Edge Apps, so i can open them as if they where their own thing

The old projects felt exactly like that because they had no top bar or anything, just the board, i just saved them while in full screen

**Comment by wellemut** [comment]
Yes the full screen mode on boards in the new project mode would be really helpful! There is just too much header and Dropdowns while working on the issues every day!

Bring back the fullscreen-Button from the classic boards!
![grafik](https://user-images.githubusercontent.com/15019030/187248238-e259d338-d920-4660-b892-6bc8c3665cc3.png)

**Comment by rileybroughten** [comment, marked as answer]
👋 Thank you for your feedback! I'm going to close this in favor of the original [discussion](https://github.com/orgs/community/discussions/5749) post to track all feedback and updates in one place!

**Reply by RalfJung** [reply]
Nice to have it all in one place, so that it can then be completely ignored in that place.

Btw there is another duplicate at https://github.com/orgs/community/discussions/10033.
